File: maxArray.js
// Encontrar maior número do array
// Sem usar Math.max

const maxArray = (array) => {
    let maxValue = array[0]

    for (let value of array) {
        if (value > maxValue) { maxValue = value }
    }

    return maxValue
}

console.log(maxArray([10, 20, 30, 50, 40]))

File: python/max_array.py
# Encontrar maior número do array
# Sem usar Math.max

def max_array(array):
    max = 0
    for item in array:
        if item > max:
            max = item

    print(max)

max_array([3, 2, 5, 7, 9, 4, 1])